README.md

```diff
@@ -354,20 +344,108 @@ The CLI accepts a series of files or globs to be formatted, as well as a set of
+Options can also be passed as environment variables.
+
+```
+FORMAT_PACKAGE_VERBOSE=true
+```
+
+| **Option** | **Alias** | **ENV**                | **Description**                                                                                                                                                                                                               | **Default**                  |
+| ---------- | --------- | ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- |
+| `config`   | `c`       | FORMAT_PACKAGE_CONFIG  | Path to a custom configuration to use. This configuration can be JavaScript, `JSON`, or any other format that your configuration of node can `require`. The default configuration can be found [here](lib/defaults/index.js). |                              |
+| `write`    | `w`       | FORMAT_PACKAGE_WRITE   | Write the output to the location of the found `package.json`                                                                                                                                                                  | **false**                    |
+| `ignore`   | `i`       | FORMAT_PACKAGE_IGNORE  | Patterns for ignoring matching files                                                                                                                                                                                          | **`['**/node_modules/**']`** |
+| `verbose`  | `v`       | FORMAT_PACKAGE_VERBOSE | Print the output of the formatting                                                                                                                                                                                            | **false**                    |
+| `help`     | `h`       |                        | Print help menu                                                                                                                                                                                                               |                              |
 
 You can also see the available options in the terminal by running:
 
 ```
 yarn format-package --help
 ```
 
+## Config
+
+`format-package` supports a custom configuration from:
+
+- a path specified with `--config`
+- a config file relative to `process.cwd()`
+- a path from an environment variable `FORMAT_PACKAGE_CONFIG`
+
+### with `--config`
+
+When using `--config` for custom configuration, this can be a module name
+or a file relative to `process.cwd()`.
+
+```
+yarn format-package -c @acme/format-config package.json
+yarn format-package -c ./configs/format-config.js package.json
+yarn format-package -c format-config.js package.json
+```
+
+### with a config file
+
```

Comment:
Can we add some explanation here that the package will automatically look for these files, in a numbered order, if no `--config` option is passed? Maybe something like:

If no `--config` option is provided, the package will automatically look for a defined configuration, proceeding down an ordered lookup list until one is found:
1. `format-package` property in `package.json`
1. `format-package.json`
1. `format-package.js`
1. `format-package.yaml`

Each type of config file is expected to export a config object in the language denoted by their file extension.